lbrynet/wallet/manager.py

```diff
@@ -264,7 +264,12 @@ def tip_claim(self, amount, claim_id, account):
     @defer.inlineCallbacks
     def abandon_claim(self, claim_id, txid, nout):
         account = self.default_account
-        claim = yield account.get_claim(claim_id)
+        if claim_id is not None:
+            claim = yield account.get_claim(claim_id=claim_id)
+        elif txid is not None and nout is not None:
+            claim = yield account.get_claim(txid=txid, nout=nout)
+        else:
+            raise Exception('Must specify claim_id, or txid and nout')
```

Comment:
No check for claim being null, should raise an error that claim was not found.

Also, you can reduce the if statements to just: `account.get_claim(claim_id=claim_id, txid=txid, nout=nout)`. Would have the same result but much less code.